src/transport_picoquic.cpp

```diff
@@ -269,16 +270,10 @@ pq_loop_cb(picoquic_quic_t* quic, picoquic_packet_loop_cb_enum cb_mode, void* ca
 
                 packet_loop_time_check_arg_t* targ = static_cast<packet_loop_time_check_arg_t*>(callback_arg);
 
-                /*
-                log_msg << "packet_loop_time_check time: "
-                    << targ->current_time << " delta: " << targ->delta_t;
-                transport->logger.log(LogLevel::info, log_msg.str());
-                */
-
                 // TODO: Add config to set this value. This will change the loop select
                 //   wait time to delta value in microseconds. Default is <= 10 seconds
-                if (targ->delta_t > 1000) {
-                    targ->delta_t = 1000;
+                if (targ->delta_t > 5000) {
+                    targ->delta_t = 5000;
```

Comment:
Magic numbers again. Also, I asked this in a different PR as well, but what is `delta_t`?

Reply:
Oh, I see the comment above now, it looked like part of the TODO. "delta value in microseconds". Delta of... since we last waited?

Reply:
It's the FD select delay in the picoquic loop.  It defaults to be a calculation from last time/wake but we are overriding it by setting it to a specific value if it's over it. If we didn't set this, it would seriously delay things when there isn't data being read. Need to add config for this too.  It's a reasonable number, but we might want to change it per client/server.